README.md

```diff
@@ -6,27 +6,27 @@
 
 - [What is Rook?](#what-is-rook)
 - [Status](#status)
-- [Kubernetes](#kubernetes)
-- [Rook Standalone Service](#rook-standalone-service)
-- [Block, File and Object Storage](#block-file-and-object-storage)
+- [Quickstart Guides](#quickstart-guides)
 - [Building](#building)
-- [Design](#design)
 - [Contributing](#contributing)
 - [Contact](#contact)
 - [Licensing](#licensing)
 
 ## What is Rook?
 
 Rook is a distributed storage system designed for cloud native applications. It aims to be the storage solution for container-native app developers. 
-Just like your container apps, Rook is designed to take advantage of the resource management and orchestration of the Kubernetes platform.
-This allows Rook to share cluster resources with your apps, allowing it to dynamically scale as your apps scale and save the effort to manage storage infrastructure separately. 
-Rook provides persistent storage for your Kubernetes apps - which is one of the largest barriers for developers to fully containerize their apps. 
-Rook exposes file, block, and object storage on top of shared resource pools. Rook has minimal
-dependencies and can be deployed in dedicated storage clusters or converged clusters. It's
+Just like your container apps, Rook is designed to take advantage of the resource management by orchestration platforms, such as Kubernetes.
+This allows Rook to share cluster resources with your apps, allowing it to dynamically scale as your apps scale and save the effort to manage storage infrastructure separately.  
+
+Rook provides persistent storage for your Kubernetes apps - which is one of the largest barriers for developers to fully containerize their apps.
+Rook exposes file, block, and object storage on top of shared resource pools.  
+
+Rook has minimal dependencies and can be deployed in dedicated storage clusters or converged clusters. It's
 self-managing, self-protecting, self-healing, and is designed to just work without teams of
-engineers managing it. It scales from a single node, to multi-PB clusters spread geographically.
-It's based on the [Ceph](http://ceph.com) project with over 10 years of production deployments in some of the
-largest storage clusters in the world.
+engineers managing it. It scales from a single node, to multi-PB clusters spread geographically.  
+
+It is based on the [Ceph](http://ceph.com) project that has over 10 years of production deployments in some of the
+largest storage clusters in the world.  
 
```

Comment:
I recommend we revert back to the original abstract and add a Kubernetes section.

```Rook is a distributed storage system designed for cloud native applications. It exposes file, block, and object storage on top of shared resource pools. Rook has minimal dependencies and can be deployed in dedicated storage clusters or converged clusters. It's self-managing, self-protecting, self-healing, and is designed to just work without teams of engineers managing it. It scales from a single node, to multi-PB clusters spread geographically. It's based on the Ceph project with over 10 years of production deployments in some of the largest storage clusters in the world.

Rook integrates deeply into popular container environments like Kubernetes and leverages facilities for lifecycle management, resource management, scale-out and upgrades. Rook also integrates into the Kubernetes API to expose a uniform surface area for management.
```